src/resizing.jl

```diff
@@ -168,8 +168,16 @@ function restrict_indices(r::UnitRange)
 end
 
 # imresize
-imresize(original::AbstractArray, dim1, dimN...) = imresize(original, (dim1,dimN...))
+imresize(original::AbstractArray, dim1::T, dimN::T...) where T<:Union{Integer,AbstractUnitRange} = imresize(original, (dim1,dimN...))
+imresize(original::AbstractArray, short_size::AbstractArray{T,1}) where T <:Union{Integer,AbstractUnitRange} = imresize(original, (short_size...))
```

Comment:
I'm starting to think, do we really need to support `Array` type size, since size in Julia is  conventionally `Tuple` type

Reply:
I don't think we need to support Array, a `MethodError` seems sufficient.

Reply:
i would argue no. there is little upside of supporting a suboptimal interface if it has no upsides.

Once thing that to me seems pretty useful is to support it marked as deprecation (but permanently). This would enable newcomers that are used to a different interface to quickly adapt